install.sh

```diff
@@ -33,11 +33,15 @@ chmod +x /usr/local/sbin/detect.sh
 cp packet_filter.py /usr/local/sbin
 cp rids.py /usr/local/sbin
 
-# move sysctl .service config to /etc/systemd and start service
-cp detection.service /etc/systemd/system/
+# Define sysctl .service config to /etc/systemd and start service
+
+# first, stop the service if it exists and is running
+systemctl stop rids_detection.service >& /dev/null
+
+cp rids_detection.service /etc/systemd/system/
 systemctl daemon-reload
-systemctl start detection.service
-systemctl enable detection.service
+systemctl start rids_detection.service
```

Comment:
The name is stuttering now. "RIDS" already means detection.

Reply:
I can make it just rids.service, but I think including the term 'detection' gives more of a clue if someone is looking at the list of service names or reading over the journalctl logs.  With rids.service, I worried it's not as obvious unless you remember what rids means.  With detection.service I worried that the name is too vague.  Probably, though, many of the log statements from the service will make it clear; I'll remove _detection from the name.